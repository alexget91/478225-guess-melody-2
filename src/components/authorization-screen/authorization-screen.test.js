import React from "react";
import renderer from "react-test-renderer";
import AuthorizationScreen from "./authorization-screen";

it(`Authorization screen correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<AuthorizationScreen onFormSubmit={jest.fn()}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
