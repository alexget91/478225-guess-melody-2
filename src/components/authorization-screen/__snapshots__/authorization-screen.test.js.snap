// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authorization screen correctly renders after relaunch 1`] = `
<section
  className="login"
>
  <div
    className="login__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="login__title"
  >
    Вы настоящий меломан!
  </h2>
  <p
    className="login__total"
  >
    За 3 минуты и 25 секунд вы набрали 12 баллов (8 быстрых), совершив 3 ошибки
  </p>
  <p
    className="login__text"
  >
    Хотите сравнить свой результат с предыдущими попытками? Представьтесь!
  </p>
  <form
    action=""
    className="login__form js-login-form"
    onSubmit={[Function]}
  >
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="name"
      >
        Логин
      </label>
      <input
        className="login__input js-email-input"
        id="name"
        name="name"
        required={true}
        type="email"
      />
    </p>
    <p
      className="login__field"
    >
      <label
        className="login__label"
        htmlFor="password"
      >
        Пароль
      </label>
      <input
        className="login__input js-password-input"
        id="password"
        name="password"
        required={true}
        type="password"
      />
      <span
        className="login__error"
      >
        Неверный пароль
      </span>
    </p>
    <button
      className="login__button button"
      type="submit"
    >
      Войти
    </button>
  </form>
  <button
    className="replay"
    type="button"
  >
    Сыграть ещё раз
  </button>
</section>
`;
